lib/coinbase/webhook.rb

```diff
@@ -0,0 +1,164 @@
+# frozen_string_literal: true
+
+module Coinbase
+  # A representation of a Webhook.
+  # This class provides methods to create, list, update, and delete webhooks
+  # that are used to receive notifications of specific events.
+  class Webhook
+    class << self
+      # Creates a new webhook for a specified network.
+      #
+      # @param network_id [String] The network ID for which the webhook is created.
+      # @param notification_uri [String] The URI where notifications should be sent.
+      # @param event_type [String] The type of event for the webhook (e.g., erc20_transfer).
```

Comment:
Is this enumerated anywhere?

Could we reflect that it should be a valid webhook event type:
https://github.com/coinbase/coinbase-sdk-ruby/pull/124/files#diff-95cb7bee0501e4a1f7328bc2b768d9be9183831509da6768d3b100269eb48190R22-R24

Reply:
sure, reflected with current support types.